test/parallel/test-stream2-unpipe-drain.js

```diff
@@ -55,6 +51,6 @@ src1.once('readable', function() {
 
 
 process.on('exit', function() {
-  assert.equal(src1.reads, 2);
-  assert.equal(src2.reads, 2);
+  common.mustCall(src1._read, 2);
```

Comment:
@storytimesolutions If you want to make this land-able quickly, you can remove the `common.mustCall()` additions and restore the `assert.equal()` statements but use `assert.strictEqual()` instead.

In thie particular case, the `common.mustCall()` approach (which is generally preferred) is a bit trickier than it usually is.